source/administration/product-recipes-template.rst

```diff
@@ -0,0 +1,60 @@
+Product Recipe Template
+========================
+
+This template defines how a product recipe can be added for the depending
+people.
+
+Problem Statement
+-------------------
+
+Define the problem statement. The problem statement could be anything from adding
+an architecture setup or installations.
+
+The problem statement should be written from a user perspective and from an organization
+perspective.
+
+The user perspective might include:
+
+- The requirements of the user
+- The environment of the user
+- The dependencies the user has
+- The use case the user would run the installation in
+
+The organization perspective might include:
+
+- How much effort the organization adds in implementing the solution
+- How many work hours are put on implementing the solution for the user
+- Listing team putting the solution for the user
+- Dependencies with ongoing installations
+
+Pre-Installation/Pre-Configuration Notes
+----------------------------------------
+
+Define the pre-installation information. Breifly describe the pre-installation
+or pre-configuration notes. Things you might include
+
+- Environment information
+- Version information
+- Release information
+- Dependency information
+- Rollback information
+- Line of Business information
+
+Instructions Setup and Configuring the Solution
+-----------------------------------------------
+
+Define the setup instructions for the implementation that you are running.
+The instructions should be detailed and have everything about setting up and
+implementing the solution. Things you might include
+
+- Installation instructions
+- Troubleshooting instructions
+- Testing the installation
+- Testing the installation in multiple environments
+
+Defining Recipe Category For Search and Retrieval
+------------------------------------------------
+
+Define the categories and tag for the users. For enabling the users of product
```

Comment:
"For enabling the users of product recipes easily search and retrieve the recipe."

This sentence is a little unclear and could also be combined with the first sentence.